config/manager/kustomization.yaml

```
@@ -13,4 +13,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: tetrate/kubegres
-  newTag: v1.16.0-tetrate-v21
+  newTag: v1.16.0-tetrate-v23
```

```suggestion
  newTag: v1.16.0-tetrate-v21
```
Let's bump it later, when all PRs are merged.